src/xmipp/legacy/libraries/tomo/tomo_tiltseries_detect_misalignment_corr.h

```diff
@@ -0,0 +1,164 @@
+/***************************************************************************
+ *
+ * Authors:    Federico P. de Isidro Gomez			  fp.deisidro@cnb.csic.es
+ *
+ * Unidad de  Bioinformatica of Centro Nacional de Biotecnologia , CSIC
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA
+ * 02111-1307  USA
+ *
+ *  All comments concerning this program package may be sent to the
+ *  e-mail address 'xmipp@cnb.csic.es'
+ ***************************************************************************/
+
+#ifndef __TOMO_TS_DETECT_MISALI_CORR
+#define __TOMO_TS_DETECT_MISALI_CORR
+
+#include <iostream>
+#include <core/xmipp_program.h>
+#include <core/xmipp_image.h>
+#include <core/metadata_vec.h>
+#include "core/xmipp_image_generic.h"
+#include <core/xmipp_fft.h>
+#include <core/xmipp_fftw.h>
+#include <math.h>
+#include <limits>
+#include <complex>
+#include <data/fourier_filter.h>
+#include <string>
+#include <data/point3D.h>
+#include <data/point2D.h>
+#include <stdio.h>
+#include <fstream>
+#include <core/transformations.h>
+
+#define VERBOSE_OUTPUT
+
+// #define DEBUG_DIM
```

Comment:
remove the debugging comments

Reply:
This is commented on since it is only useful for debugging. In production, it is better to keep them commented to improve performance. It is better to keep debugging code since this is legacy and will be useful in future developments.